packages/strapi-plugin-i18n/config/functions/register.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+const { getService } = require('../../utils');
+
+module.exports = () => {
+  Object.values(strapi.models).forEach(model => {
+    if (getService('content-types').isLocalized(model)) {
+      _.set(model.attributes, 'localizations', {
+        writable: true,
+        private: false,
+        configurable: false,
+        type: 'json',
+      });
+
+      _.set(model.attributes, 'locale', {
+        writable: true,
+        private: false,
+        configurable: false,
+        type: 'string',
+      });
+    }
+  });
```

Comment:
I guess it is here for the moment as it is the fastest way. However do we agree it will move in a better place in V4?

Reply:
I don't understand the question